weepickle/src/main/scala-3/com/rallyhealth/weepickle/v1/LowPriorityImplicits.scala

```diff
@@ -40,16 +40,31 @@ abstract class LowPriorityImplicits extends AttributeTagged {
     }
   }
 
-  private def fromEnumerationName[E <: scala.Enumeration, V <: scala.Enumeration#Value](e: E)(implicit ev: V =:= e.Value): From[V] = {
+  private def fromEnumerationName[E <: scala.Enumeration, V <: scala.Enumeration#Value](e: E)(
+    implicit ev: V =:= e.Value): From[V] = {
     val cache = new ConcurrentHashMap[V, String].asScala // mitigate toString() slowness.
     FromString.comap(v => cache.getOrElseUpdate(v, v.toString))
   }
 
-  def fromToEnumerationName[E <: scala.Enumeration, V <: scala.Enumeration#Value](e: E)(implicit ev: V =:= e.Value): FromTo[V] = {
+  def fromToEnumerationName[E <: scala.Enumeration, V <: scala.Enumeration#Value](e: E)(
+    implicit ev: V =:= e.Value): FromTo[V] = {
     FromTo.join(toEnumerationName(e), fromEnumerationName(e))
   }
 
-  def fromToEnumerationId[E <: scala.Enumeration, V <: scala.Enumeration#Value](e: E)(implicit ev: V =:= e.Value): FromTo[V] = {
+  def fromToEnumerationId[E <: scala.Enumeration, V <: scala.Enumeration#Value](e: E)(
+    implicit ev: V =:= e.Value): FromTo[V] = {
     FromTo.join(ToInt.map(e.apply(_).asInstanceOf[V]), FromInt.comap(_.id))
   }
+
+  //  Native Scala 3 enums extend scala.reflect.Enum transparently.
+  //  By name encoding provided in default derives, so only id encoding provided here.
+  // TODO: do we even need this? If we do, should it be here (it isn't implicit)?
```

Comment:
What do you think @htmldoug ?

Reply:
I only see one place that `fromToEnumerationId` is used in our current application code and I'm not sure it's even intentional. I'd be fine with leaving this out. Users can always handle with custom picklers.

Reply:
That's kind of where my head was at too. I'll remove it and change the unit test to construct it locally.